.travis.yml

```diff
@@ -6,7 +6,7 @@ matrix:
     - python: "3.6-dev"
       env: TEST_CMD="flake8"
     - python: "3.5"
-      env: TEST_CMD="./tests/mypy_test.py"
+      env: TEST_CMD="./tests/mypy_test.py && ./tests/mypy_selftest.py"
```

Comment:
How about we spin up another job for this? We could cover a release Python version with this (e.g. "3.6"). Then you wouldn't need `bash -c` below.